cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release-pipeline-sa.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: release-service-account
```

Comment:
we should include the RemoteSecrets as shown here:

https://github.com/redhat-appstudio/tenants-config/blob/main/cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/remote-secrets/remote_secret-quay-token.yaml

Reply:
done